avaframe/in3Utils/fileHandlerUtils.py

```diff
@@ -283,6 +283,7 @@ def getFilterDict(cfg, section):
 def splitIniValueToArraySteps(cfgValues, returnList=False):
     """ read values in ini file and return numpy array or list if the items are strings;
         values can either be separated by | or provided in start:end:numberOfSteps format
+        if separated by : also optional add one additional value using & 
```

Comment:
This also needs to be added to the documentation

Reply:
added to the doc ' - a single value can be added by appending &4.0 for example'